index.js

```diff
@@ -1,5 +1,6 @@
 const express = require('express');
 const path = require('path');
+require('dotenv').config();
```

Comment:
this dotenv initialization does not need to be here, you've already done this in the webpack config

Reply:
@Darthrighteous done